src/app/containers/RadioPageBlocks/Blocks/LiveRadio/index.jsx

```diff
@@ -61,6 +61,7 @@ const LiveRadioContainer = ({ idAttr, externalId, id }) => {
           placeholderSrc={isAmp ? liveRadioPlaceholderImageSrc : null}
           showPlaceholder={!isAmp ? false : null}
           src={embedSource}
+          title={translations.mediaAssetPage.audioPlayer}
```

Comment:
```suggestion
          path(['mediaAssetPage', 'audioPlayer'], translations);
```

And import path into this file